builders/ethereum/dev-env/thirdweb.md

```diff
@@ -110,35 +112,47 @@ thirdweb offers SDKs for a range of programming languages, such as React, React
 2. Input your preferences for the command line prompts:
 
     1. Give your project a name
-    2. Choose your network. You can choose **EVM** for Moonbeam
-    3. Choose your preferred framework: **Next.js**, **Create React App**, **Vite**, **React Native**, **Node.js**, or **Express**. For this example, you can select **Create React App**
-    4. Choose your preferred language: **JavaScript** or **TypeScript**
+    2. Choose your preferred framework: **Next.js**, **Vite**, or **React Native**. For this example, select **Vite**
 
 3. Use the React or TypeScript SDK to interact with your application’s functions. This will be covered in the following section on interacting with a contract
 
+### Specify Client ID {: #specify-client-id }
+
+Before you launch your dApp (locally or publicly deployed), you must have a ThirdWeb Client ID associated with your project. A ThirdWeb Client ID is synonymous with an API key. You can create a free API key by [signing into your ThirdWeb Account and navigating to Settings -> API Keys](https://thirdweb.com/create-api-key){target=\_blank}.
```

Comment:
settings and API keys are both UI elements, so those should be in bold and can you replace `->` with "then click on" or something similar